docs/topics/db/sql.txt

```diff
@@ -403,6 +403,31 @@ is equivalent to::
     finally:
         c.close()
 
+Async Connections and cursors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+On backends that support async-native connections, you can request an async
+cursor::
+
+    from django.db import new_connection
+
+    async with new_connection() as connection:
+        async with connection.acursor() as c:
+            await c.aexecute(...)
+
+Async cursors provide the following methods:
```

Comment:
I think we're missing tests for:
* ``.aexecutemany()``
* ``.afetchone()``
* ``.afetchall()``
* ``.acopy()``
* ``.astream()``
* ``.ascroll()``

Reply:
I've added the test